SM/configs/sqlc/query.sql

```diff
@@ -1,77 +1,77 @@
---name: CreateUser: one
+-- name: CreateUser :one
 INSERT INTO Users(
     id, bitrixid, name, role 
 ) VALUES (
     $1, $2, $3, $4
 )
 RETURNING *;
--- name: DeleteUser: exec
+-- name: DeleteUser :exec
 DELETE FROM Users
 WHERE id = $1;
---name: ChangeUserRole: exec
+-- name: ChangeUserRole :exec
 UPDATE Users
 SET 
-    role = $2,
+    role = $2
 WHERE id = $1;
---name: UsersList: many
+-- name: UsersList :many
 Select * FROM Users
 ORDER BY id;
---name: UsersListByRole: many
+-- name: UsersListByRole :many
 Select * FROM Users
 WHERE role = $1
 ORDER BY id;
 
 
---name: CreateShift: one
+-- name: CreateShift :one
```

Comment:
you can create differnet files for different entities - queries could become really long!